apps.json

```diff
@@ -2939,6 +2939,12 @@
         "state": "working",
         "url": "https://github.com/YunoHost-Apps/nomad_ynh"
     },
+    "ntfy": {
+        "category": "dev",
+        "state": "working",
+        "url": "https://github.com/l-x/ntfy_ynh.git",
```

Comment:
```suggestion
        "url": "https://github.com/YunoHost-Apps/ntfy_ynh",
```

Reply:
Does that change make sense without changing state to `notworking`?

Reply:
we can merge this PR once the app is working